net/basicstation/patches/000-include_sys_time.patch

```diff
@@ -0,0 +1,11 @@
+--- a/src/aio.c
```

Comment:
Did you try to upstream this patch?

Reply:
Thanks for your review! I have opened a pull request for this patch: https://github.com/lorabasics/basicstation/pull/165